lib/containers/bats.pm

```diff
@@ -0,0 +1,71 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Common functions for BATS test suites
+# Maintainer: qa-c@suse.de
+
+package containers::utils;
+
+use base Exporter;
+use Exporter;
+
+use base "consoletest";
+use testapi;
+use Utils::Architectures;
+use utils;
+use strict;
+use warnings;
+use version_utils;
+use registration qw(add_suseconnect_product get_addon_fullname);
+
+our @EXPORT = qw(install_bats add_packagehub remove_mounts_conf switch_to_user);
+
+sub install_bats {
+    my $bats_version = "1.11.0";
```

Comment:
This will be a problem in the future as the version won't stay the same

Reply:
I'm more afraid that this version will become SLES-version dependent.

Reply:
> for now yes, but at some point it will be updated. Sooner or later and we will have this version hardcoded.

We don't need the latest version but the latest working version like we already do with lots of versioned bits in lots of places.

What's the alternative?  A master branch could bring lots of trouble.  Packaging it would be overkill and we would have to wait as discussed in Slack.

Reply:
Yes, the latest working version might change, so at least an openqa user/test can pull this value from openQA test suite variables. As of now, 1.11.0 can stay as default

Reply:
> Yes, the latest working version might change, so at least an openqa user/test can pull this value from openQA test suite variables. As of now, 1.11.0 can stay as default

Now I get what you mean.  Yes, I can parametrize this.

Reply:
Fixed!